src/app/page.tsx

```diff
@@ -1,9 +1,154 @@
-import styles from './page.module.css';
+'use client';
+
+import COLORS from '@/styles/colors';
+import Image from 'next/image';
+import { H2, H3, H4 } from '../styles/text';
+import { LinkButton } from '../components/Button';
+import homepageImage from '../../public/images/HomePageImage.webp';
+import * as Styles from './styles';
+
+type Stat = {
+  number: string;
+  label: string;
+};
+
+const stats: Stat[] = [
+  { number: '181+', label: 'Matched Cases' },
+  { number: '181+', label: 'Attorneys' },
+  { number: '65+', label: 'Interpreters & Translators' },
+  { number: '46+', label: 'Successful Outcomes' },
+];
+
+const renderStat = (stat: Stat) => (
+  <Styles.StatContainer key={stat.label}>
+    <H2 $color={COLORS.blueDark}>{stat.number}</H2>
+    <Styles.StatLabel>{stat.label}</Styles.StatLabel>
+  </Styles.StatContainer>
+);
+
+type Service = {
+  iconSrc: string;
+  title: string;
+  description: string;
+};
+
+const services: Service[] = [
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Case Assistance',
+    description:
+      '<b>Attorneys</b> licensed in any state with or without immigration law experience can represent immigrants and asylum-seekers in removal proceeding before the Immigration Court, the Board of Immigration Appeals and the U.S. Court of Appeals for the Ninth Circuit.',
+  },
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Limited Case Assignment',
+    description:
+      '<b>Law students</b> and <b>recent graduates</b> awaiting bar results can assist attorneys to screen potential clients, conduct legal and factual research and write motions and briefs in support of on-going court cases.',
+  },
+  {
+    iconSrc: 'An Icon will be Here',
+    title: 'Language Support',
+    description:
+      '<b>Interpreters</b> and <b>translators</b> can conduct volunteer translation and live interpertation. We have frequent need for individuals who speak Creole, Portuguese, Spanish, French, Arabic, and Russian',
+  },
+];
+
+const renderService = (service: Service) => (
+  <Styles.ServiceContainer key={service.title}>
+    <Styles.ServiceIcon src={service.iconSrc} alt={service.title} />
+    <H3 $color={COLORS.blueMid}>{service.title}</H3>
+    <H4
+      $color={COLORS.greyDark}
+      $fontWeight="400"
+      dangerouslySetInnerHTML={{ __html: service.description }}
+    />
+  </Styles.ServiceContainer>
+);
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <div className={styles.description}>immigration justice project</div>
-    </main>
+    <Styles.PageContainer>
+      <Styles.TitleSection>
+        <Styles.ImageContainer>
+          <Image
+            alt="background"
+            src={homepageImage.src}
+            placeholder="blur"
+            blurDataURL={homepageImage.src}
+            quality={100}
+            layout="fill"
+            objectFit="cover"
+            fill
+          />
```

Comment:
let's remove the `objectFit="cover"` here as it's a legacy prop
instead, we can add `style={{ objectFit: 'cover' }}`